lib/fluent/plugin/in_monitor_agent.rb

```diff
@@ -341,10 +292,10 @@ def self.collect_children(pe, array=[])
     # try to match the tag and get the info from the matched output plugin
     # TODO: Support output in label
     def plugin_info_by_tag(tag, opts={})
-      matches = Engine.root_agent.event_router.instance_variable_get(:@match_rules)
+      matches = Fluent::Engine.root_agent.event_router.instance_variable_get(:@match_rules)
       matches.each { |rule|
         if rule.match?(tag)
-          if rule.collector.is_a?(Output)
+          if rule.collector.is_a?(Fluent::Output)
```

Comment:
`Fluent::Plugin::Output` looks correct instead of `Fluent::Output`(this means v0.12 plugin instances only).